### ISSUE TITLE
280: Laminar Research Directives are broken for Collections 

### ISSUE THREAD

**Issue by tngreene** [X-Plane/XPlane2Blender#510]
OBJECT_OT_add_xplane_export_path_directive assumes a collection. We need to make a COLLECTION_OT version.

Related to #450

**Comment by tngreene** [comment]
While e67cf510379c6ffdf0472bde5b8ebf2d885fb726 allows new Blender 2.80 projects to use Collections, the updater's property group copy API is broken.

```
for source_member, dest_member in zip(source_members, dest_members):
```
if source_members (a collection of say 4 LR directives) and dest_members (0, of course) then nothing is copied. This API was developed for two collections that were always going to be the same length. See #493 also for when we'll fix this.

The fix is simple: while len(source_members) > dest_members.add(). If we have to remove any by that point, something is wrong. dest_members should be empty or perfect from other update utils.

**Comment by tngreene** [comment]
Was fixed with 1b176403e9
